src/kaltura-player.js

```diff
@@ -52,12 +52,14 @@ class KalturaPlayer extends FakeEventTarget {
   _firstPlay: boolean = true;
   _sourceSelected: boolean = false;
   _pluginReadinessMiddleware: PluginReadinessMiddleware;
+  _pluginsConfigure: PluginsConfigure;
 
-  constructor(options: KPOptionsObject) {
+  constructor(options: KPOptionsObject, pluginsConfigure: PluginsConfigure) {
```

Comment:
Why as an argument and not as class member that you are initialize internally?

Reply:
we already initialize the PluginsConfigure in `setup`

Reply:
so don't, initialize it here

Reply:
I can't. see https://github.com/kaltura/kaltura-player-js/blob/FEC-10732/src/setup.js#L33

Reply:
Again, you can do it inside the constructor.

Reply:
you are right. done.